concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-857	857	Pfeife	Pfeife	whistle	3107	WHISTLE (INSTRUMENT)
+Steinthal-1875-1565-858	858	Saiteninstrument	Saiteninstrument	string instrument		
+Steinthal-1875-1565-859	859	tanzen	tanzen	to dance	1879	DANCE
+Steinthal-1875-1565-860	860	singen	singen	to sing	1261	SING
+Steinthal-1875-1565-861	861	spielen	spielen	to play	1413	PLAY
+Steinthal-1875-1565-862	862	trommeln	trommeln	to drum	111	DRUMMING
+Steinthal-1875-1565-863	863	blasen	blasen	to blow	176	BLOW (WITH MOUTH)
+Steinthal-1875-1565-864	864	erzählen	erzählen	to tell	1711	TELL
+Steinthal-1875-1565-865	865	Geschichte	Geschichte	story	2060	STORY
+Steinthal-1875-1565-866	866	Fabel	Fabel	fable		
+Steinthal-1875-1565-867	867	plaudern	plaudern	to chat	2903	CHAT (WITH SOMEBODY)
+Steinthal-1875-1565-868	868	kaufen	kaufen	to buy	1869	BUY
+Steinthal-1875-1565-869	869	verkaufen	verkaufen	to sell	1571	SELL
+Steinthal-1875-1565-870	870	Handel treiben	Handel treiben	to trade	2135	TRADE OR BARTER
+Steinthal-1875-1565-871	871	Gewerbe	Gewerbe	trade, industry		
+Steinthal-1875-1565-872	872	verdienen	verdienen	to earn	733	EARN
+Steinthal-1875-1565-873	873	ehrlich	ehrlich	honest	1120	HONEST
+Steinthal-1875-1565-874	874	betrügen	betrügen	to betray	1902	BETRAY
+Steinthal-1875-1565-875	875	Waare	Waare	ware	2957	WARE
+Steinthal-1875-1565-876	876	Markt	Markt	market	633	MARKET
+Steinthal-1875-1565-877	877	Preis	Preis	price	657	PRICE
+Steinthal-1875-1565-878	878	Geld	Geld	money	1559	MONEY
+Steinthal-1875-1565-879	879	theuer	theuer	expensive	1426	EXPENSIVE
+Steinthal-1875-1565-880	880	billig	billig	cheap	1887	CHEAP
+Steinthal-1875-1565-881	881	Lohn	Lohn	wage	1122	WAGES
+Steinthal-1875-1565-882	882	Pfand	Pfand	deposit		
+Steinthal-1875-1565-883	883	bezahlen	bezahlen	to pay	718	PAY
+Steinthal-1875-1565-884	884	Verdienst	Verdienst	earnings		
+Steinthal-1875-1565-885	885	Verlust	Verlust	loss		
+Steinthal-1875-1565-886	886	schenken	schenken	to gift		
+Steinthal-1875-1565-887	887	borgen	borgen	to borrow 	1823	BORROW
+Steinthal-1875-1565-888	888	miethen	miethen	to rent	3213	HIRE OR RENT
+Steinthal-1875-1565-889	889	leihen	leihen	to lend	582	LEND
+Steinthal-1875-1565-890	890	tauschen	tauschen	to exchange	2509	EXCHANGE
+Steinthal-1875-1565-891	891	Ich bin ihm schuldig.	Ich bin ihm schuldig.	I owe him.	369	OWE
+Steinthal-1875-1565-892	892	Ich bekomme zum Geschenk.	Ich bekomme zum Geschenk.	I receive something as a present.		
+Steinthal-1875-1565-893	893	wiedergeben	wiedergeben	to return something		
+Steinthal-1875-1565-894	894	reich	reich	rich	712	RICH
+Steinthal-1875-1565-895	895	arm	arm	poor	1674	POOR
+Steinthal-1875-1565-896	896	zählen	zählen	to count	1420	COUNT
+Steinthal-1875-1565-897	897	Häuptling	Häuptling	chieftain	301	CHIEFTAIN
+Steinthal-1875-1565-898	898	König, Königin, deren Sohn	König	king	1508	KING
+Steinthal-1875-1565-899	899	König, Königin, deren Sohn	Königin	queen	1306	QUEEN
+Steinthal-1875-1565-900	900	König, Königin, deren Sohn	Prinz	prince	2554	PRINCE (RULER)
```

Comment:
PRINCE (RULER) is the other meaning of prince. For this meaning we don't have a concept.